recipes/openfe-analysis/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "openfe-analysis" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openfreeenergy/openfe_analysis/archive/{{ version }}.tar.gz
+  sha256: 74afe903ef9e9095a59b48ee5e06a6f903d10cd693cb19ca2f0f3d7c8f2fe80e
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py==38] 
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+    - versioningit
+  run:
+    - MDAnalysis
+    - click
+    - netCDF4
+    - openff-units
+    - python
+
+test:
+  imports:
+    - openfe_analysis
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/OpenFreeEnergy/openfe_analysis
+  summary: 'Analysis tools for the openfe ecosystem'
+  description: |
+    simplejson is a simple, fast, complete, correct and extensibledd
```

Comment:
@mikemhenry oi

Reply:
goodness me I thought this was going to be the one I get right in a commit